src/instance.js

```diff
@@ -94,12 +95,16 @@ export default class Instance {
 		this.reconciler.updateContainer(tree, this.container);
 	}
 
-	unmount() {
+	unmount(error) {
```

Comment:
This needs to be documented.

Reply:
Done.

Reply:
Actually I think unmounting with a custom error doesn't really make sense, it's only related to `exit()`. It's only useful for internal purposes. I think it should be removed from docs and convert this https://github.com/vadimdemedes/ink/blob/07ff24bd0a15adacad75cf04d7beb33bb733a2d2/src/render.js#L34 to `unmount: () => instance.unmount()`?